egs/ifnenit/v1/RESULTS.txt

```diff
@@ -0,0 +1,3 @@
+WER%                    abc-d       abcd-e      abcde-f     abcde-s 
+cnn1a                   1.46%       4.18%       5.79%       9.15%
+cnn_chainali_1b         1.96%       3.28%       4.80%       7.63%
```

Comment:
Could you please update the results (after your changes)?

Reply:
Actually the `diff.txt` file was empty. So every test words are in the train set. I've decoded `abc-e` version and the result was the same. I think result would be the same for other versions too.